plugin.cpp

```diff
@@ -36,35 +36,52 @@ static const char *default_config = QUOTE({
 			"type": "boolean",
 			"displayName": "Enabled",
 			"default": "false",
-			"order" : "5"
+			"order" : "7"
 		       	},
+        "toleranceMeasure": {
+			"description": "Whether tolerance is specified as a percentage or an absolute value",
+			"type": "enumeration",
+			"options" : [ "percentage", "absolute value" ],
```

Comment:
Change case of option text to match with others.

Reply:
ok sure.